### DISCUSSION TITLE
-slicemesh forced close even though the loop is not finished.

### DISCUSSION THREAD

**Discussion by nippyawn** [neperfepx/neper#886] [Q&A] [answered]
Hello

When I run under code , program forced close even though the loop is not finished.
I want to run loop from 150 to 370 , but it stopped in the middle loop. (for example 200 loop or something like that)
Please tell me how to fix it.

```
neper -V model.msh -showelset "file(test.txt)" -dataelsetcol ori -dataelsetcolscheme ipf -cameracoo x:y:z=-800+z -cameraprojection orthographic -camerasky 0:0:1 -loop Z 150 1 370 -slicemesh "z=Z" -outdir /home... -print img_Z -endloop
```

thank you.

**Comment by rquey** [comment, marked as answer]
This discussion has been closed as obsolete. Please repost if necessary.
